nixos/modules/services/monitoring/thanos.nix

```
@@ -642,6 +646,10 @@ let
 
     receive = params.common cfg.receive // params.objstore cfg.receive // {
 
+      grpc-compression = mkParam types.str ''
```

```suggestion
      receive.grpc-compression = mkParam types.str ''
```

https://github.com/thanos-io/thanos/blob/main/cmd/thanos/receive.go#L894

`receive` command uses `receive.grpc-compression` if I am not mistaken